index.html

```diff
@@ -391,6 +384,21 @@ <h1>Type-based Extensions</h1>
     </table>
 
   </section>
+
+  <section>
+    <h1>Securing Mechanisms</h1>
+
+    <table class="simple">
+      <thead>
+        <th>Specification</th><th>Description</th>
```

Comment:
It seems like it would be useful for this table to list the media types, `proof.type`s and `proof.cryptosuite`s that each securing mechanism handles. Otherwise, if you get a VC you don't understand, you have to read all of the specifications to figure out where it's defined.

Reply:
The vc-jose-cose securing mechanism doesn't use proof at all, so at least, I wouldn't phrase it this way.

When using JOSE, the algorithms registered at https://www.iana.org/assignments/jose/jose.xhtml#web-signature-encryption-algorithms apply.  When using COSE, the algorithms registered at https://www.iana.org/assignments/cose/cose.xhtml#algorithms apply.

Reply:
Related to the question of "what version" and "what securing mechanism"... this issue on media types: https://github.com/w3c/vc-data-model/issues/1363#issuecomment-1828449697